File: src/java/ironfist/persistence/PersistenceException.java
package ironfist.persistence;

import java.io.IOException;

/**
 * @author pmgremo
 *  
 */
public class PersistenceException extends IOException {

  /**
   * Constructor for PersistenceException.
   */
  public PersistenceException() {
    super();
  }

  /**
   * Constructor for PersistenceException.
   * 
   * @param s
   */
  public PersistenceException(String s) {
    super(s);
  }

}

File: src/main/java/ukalus/persistence/PersistenceException.kt
package ukalus.persistence

import java.io.IOException

/**
 * @author pmgremo
 */
class PersistenceException
/**
 * Constructor for PersistenceException.

 * @param s
 */(s: String?) : IOException(s) {

    companion object {

        private val serialVersionUID = 3618981191634858544L
    }

}